tests/home.spec.ts

```diff
@@ -0,0 +1,38 @@
+import {test, expect} from '@playwright/test';
+
+const LOCAL_DOMAIN = `http://localhost:8080/`;
+
+test.describe('Home page', () => {
+  test('splashLogo accessible.', async ({page}) => {
+    await page.goto(LOCAL_DOMAIN);
+    expect(await page.locator('#splashLogo').getAttribute('role')).toBe(
+      'heading'
+    );
+    const homePageImg = page.locator('#splashLogo > img');
+    expect(await homePageImg.getAttribute('aria-label')).toBe('Lit');
+  });
+
+  test('homepage screenshot test', async ({page}) => {
+    await page.goto(LOCAL_DOMAIN);
+    const homePageIntro = page.locator('#intro');
+    expect(await homePageIntro.screenshot()).toMatchSnapshot('home-intro.png');
```

Comment:
There will be rendering differences between platforms right? Looks like the screenshot we have here is for mac OS. How will a developer update the linux screenshot that would be tested on CI?

Reply:
I temporarily removed screenshot tests. I think there is a way to make it work.